deployment-guidelines.md

```diff
@@ -17,3 +17,24 @@
 **Obviously the above is a rough guide and *shouldn't* replace common sense. e.g. If something happens out of hours and a developer can't get hold of a tester, they shouldn't hold off deploying a fix.**
 
 N.B. For expedited issues please refer to the following [guide](expedited-procedure.md).
+
+## Web Assets
+
+In order for new code to be deployed to our customers it first needs to be uploaded to the correct S3 bucket.
+
+A single page or app could use code from multiple S3 locations so to help simplify this we have decided on the following structure:
+
+* One S3 bucket per environment, not per project.
```

Comment:
That was the original idea, but IT agreed that this way was easier to manage now that our repos are growing. Also we can have the staging bucket on IT's staging account and production bucket on the production one.